worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitPlayer.java

```diff
@@ -52,10 +52,16 @@
 import com.sk89q.worldedit.world.block.BlockTypes;
 import com.sk89q.worldedit.world.gamemode.GameMode;
 import com.sk89q.worldedit.world.gamemode.GameModes;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
I would avoid this; either we should use a record or just Map.Entry, no need to add 3rd party dependencies wher it's not really needed (_especially_ apache commons as that can _massively_ increase the jar size)